docs/exterminate.rst

```diff
@@ -64,6 +66,7 @@ Methods
 :magma: Boil the unit in magma (not recommended for magma-safe creatures).
 :butcher: Will mark the units for butchering instead of killing them. This is
     more useful for pets than armed enemies.
+:knockout: Will put units into an unconscious state for 30k ticks.
```

Comment:
```suggestion
:knockout: Will put units into an unconscious state for 30k ticks (about a month).
```

Reply:
more testing needed for this one, I dunno if adventure mode ticks align with fortress mode ticks

Reply:
they don't. the month estimate is for fort mode. you could define either define a time span that `knockout` will last and adjust different ticks based on the mode, or else document the different time spans that `knockout` will last in fort mode vs. adventure mode.